crates/api/Cargo.toml

```diff
@@ -23,8 +23,10 @@ lemmy_db_views = { workspace = true, features = ["full"] }
 lemmy_db_views_moderator = { workspace = true, features = ["full"] }
 lemmy_db_views_actor = { workspace = true, features = ["full"] }
 lemmy_api_common = { workspace = true, features = ["full"] }
+lemmy_api_crud = { workspace = true }
```

Comment:
Dont do this, it will slow down compilation. Instead move anything that you need from api_crud into api_common.